connectors/es/client.py

```diff
@@ -176,6 +176,17 @@ async def ensure_exists(self, indices=None):
     async def delete_indices(self, indices):
         await self.client.indices.delete(index=indices, ignore_unavailable=True)
 
+    async def clean_index(self, index_name):
+        return await self.client.delete_by_query(
+            index=index_name, body={"query": {"match_all": {}}}, ignore_unavailable=True
+        )
+
+    async def list_indices(self):
+        return await self.client.indices.stats(index="search-*")
+
+    def client(self):
+        return self.client
```

Comment:
We need to have access to Elasticsearch client because not all method we need are exposed in ESclient. 

e.g. https://github.com/elastic/connectors/blob/dmitrii/5185-connectors-clients-serverless/connectors/cli/connector.py#L92

Reply:
Yeah, but you can still access this class's client over `client` property:

```python
client = EsClient()
internal_client = client.client
```

The method in discussion, however, adds one more way to access it:

```
client = EsClient()
internal_client = client.client() # it's a method, thus brackets
```

So from how it _looks_, there's just no need for this method definition, as the field is accessible by default. Or am I missing something? 🤔

Reply:
Understood, thanks. 

I think I faced the issue when `client` was inaccessible however it works right now. I'm going to remove that method.